src/material-examples/cdk-virtual-scroll-context/cdk-virtual-scroll-context-example.css

```diff
@@ -0,0 +1,13 @@
+.example-viewport {
+  height: 200px;
+  width: 200px;
+  border: 1px solid black;
+}
+
+.example-item-detail {
+  height: 18px;
+}
+
+.example-alternate {
+  background: rgba(0, 0, 0, .2);
```

Comment:
I think we may end up with white text on a white background if somebody selected a dark theme in the docs site.

Reply:
I don't think our examples really support theme styles, but I'll use a 50% gray so it at least looks ok in either.